Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/BackgroundTask/AddIshBackgroundTask.cs

```diff
@@ -115,10 +138,24 @@ public sealed class AddIshBackgroundTask : BackgroundTaskCmdlet
         [AllowEmptyCollection]
         public IshObject[] IshObject { get; set; }
 
+        /// <summary>
+        /// <para type="description">The InputDataTemplate (e.g. IshObjectWithLngRef) indicates whether a list of ishObjects or one ishObject is submitted as input data to the background task.</para>
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = false, ParameterSetName = "IshObjectsGroup")]
+        public InputDataTemplate InputDataTemplate { get; set; } // Change to enum.
```

Comment:
Yes, an enum makes sense, example from GetBackgroundTask.cs ... share an ISHRemote enum, not the API25 one
```
/// <summary>
        /// <para type="description">Enumeration indicating if only events of the current user or all events must be retrieved</para>
        /// </summary>
        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParameterGroup")]
        [ValidateNotNullOrEmpty]
        public Enumerations.UserFilter UserFilter
        {
            private get { return Enumerations.UserFilter.All; }  // required otherwise XmlDoc2CmdletDoc crashes with 'System.ArgumentException: Property Get method was not found.'
            set { _userFilter = EnumConverter.ToUserFilter<BackgroundTask25ServiceReference.eUserFilter>(value); }
        }
```